src/pushsource/_impl/backend/errata_source/errata_source.py

```diff
@@ -151,7 +151,27 @@ def _advisory_ids(self):
         return self._errata
 
     def _push_items_from_raw(self, raw):
-        erratum = ErratumPushItem._from_data(raw.advisory_cdn_metadata)
+        raw_metadata = raw.advisory_cdn_metadata.copy()
+        if raw.advisory_cdn_metadata.get("container_list"):
+            new_container_list = []
+            # for every item in container list replace build with external repos
+            for clistitem in raw_metadata["container_list"][:]:
+                new_clistitem = {}
+                for build in list(clistitem.keys()):
+                    build_repos = raw.advisory_cdn_docker_file_list[build]["docker"][
+                        "target"
+                    ]["external_repos"]
+                    val = clistitem[build]
+                    new_val = {"digest": val["digest"], "images": {}}
+                    for image, digest_dict in val["images"].items():
+                        arch = image.split(".")[-3]
```

Comment:
Minor: a comment or two here explaining what it's doing would be valuable (e.g. an example value of `image`).